src/sentry/static/sentry/app/views/incidents/list/index.tsx

```diff
@@ -147,7 +147,8 @@ class IncidentsListContainer extends React.Component<Props> {
         <PageContent>
           <PageHeader>
             <PageHeading>
-              {t('Alerts')} <BetaTag />
+              {t('Alerts')}{' '}
+              <BetaTag title={t('Alert stream only shows metric alerts while in beta')} />
```

Comment:
since alert stream is not a user-facing name, let's reword this to "This feature may change in the future, and currently only shows metric alerts"